src/platforms/unity/configuration/native-support.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Native Support
+description: "Learn how the Unity SDK handles native support."
+sidebar_order: 30
+---
+
+<Note>
+
+The native support requires configuration using the Sentry Unity Editor window. Support for programatic initialization via C# is planned.
+
+</Note>
+
+<Note>
+
+Our Unity SDK currently supports native iOS when building for device only. Support for the iOS simulators is planned.
+
+</Note>
+
+<Note>
+
+[Releases & Health](platforms/unity/configuration/releases/): The Crash Free Rate only indicates real crashes on the native layer and is currently limited to iOS.
+
+</Note>
+
+The native support is `enabled` by default. The Unity SDK passes the options defined via the Sentry Unity Editor window during build time. For Android it embeds the [Android SDK](/platforms/android/) inside the Gradle project created by Unity. The [iOS SDK](/platforms/apple/guides/ios/) gets embedded inside the generated Xcode project. This means there's no need to use the Unity [built-in crash reporting](https://docs.unity3d.com/ScriptReference/CrashReport.html) functionality.
+
+You can opt out of the native support by unchecking `iOS Native Support` and `Android Native Support` in the configuration window.
+
+### Debug Symbols
+
+Our Unity SDK does not yet support the automated upload of debug symbols, but since it's using other Sentry SDKs this can be set up manually. Check out our documentation for [iOS](/platforms/apple/guides/ios/dsym/) and [Android](/platforms/android/proguard/).
```

Comment:
```suggestion
Our Unity SDK does not yet support the automated upload of debug symbols, but since it's using other Sentry SDKs this can be set up manually. Check out our documentation for [iOS](/platforms/apple/guides/ios/dsym/) and [Android](/platforms/android/using-ndk/#symbolicate-stack-traces).
```

they are probably not using Gradle, so this page for Android would make more sense I guess

Reply:
Unity outputs a Grade project at the end